python/pyspark/sql/functions.py

```diff
@@ -426,6 +426,29 @@ def monotonically_increasing_id():
     return Column(sc._jvm.functions.monotonically_increasing_id())
 
 
+@since(2.1)
+def monotonically_increasing_id(offset):
```

Comment:
We can introduce a new method which accepts offset.
How about monotonically_increasing_id_w_offset ?